upup/pkg/fi/utils/reflect.go

```diff
@@ -211,3 +211,34 @@ func IsPrimitiveValue(v reflect.Value) bool {
 		return false
 	}
 }
+
+// FormatValue returns a string representing the value
+func FormatValue(value interface{}) string {
+	// Based on code in k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go
```

Comment:
It's buried in Error sadly.  We could reuse it, but it's probably cleaner to copy it.